gradle.properties

```diff
@@ -3,7 +3,7 @@ xhAppName=Toolbox
 xhAppPackage=io.xh.toolbox
 xhAppVersion=7.0-SNAPSHOT
 
-hoistCoreVersion=29.0-SNAPSHOT
+hoistCoreVersion=30.0-SNAPSHOT
```

Comment:
I didnt make this upate. This is what is on `develop`
https://github.com/xh/toolbox/blob/develop/gradle.properties#L6
When I merged up this appeared. I rebased origin/develop into local/develop, then merged local/develop into this branch. That has never before made a commit list which was not mine.